Minor edits to ncov tutorial (#942)
nextstrain/ncov: docs/src/tutorial/genomic-surveillance.rst

      conda activate nextstrain

> .. |today| date::

Reviewer: Can this be moved closer to the line defining `|last-month|`?

Author: Fixed in 7ea33bf

      :width: 400
      :alt: GISAID EpiCoV Search

.. |last-month| timedelta::
   :weeks: -4

2. Filter to sequences that pass the following criteria:

   1. From Idaho, USA

>    2. Collected within the last month (between |last-month| and |today| at the time of writing)

Reviewer: Reword given dynamic dates:

```suggestion
   2. Collected within the last month (e.g. |last-month| to |today|)
```

>    2. Collected within the last month (between |last-month| and |today| at the time of writing)

Reviewer: Convert `image` to `figure`, add a caption explaining the static dates:

```rst
   .. figure:: ../images/gisaid-select-sequences-idaho-highlighted.png
      :width: 700
      :alt: GISAID EpiCoV filter and select sequences

      Example of GISAID filters using collection date between 2022-03-01 and 2022-04-01
```

Author: Do you want to update this when you update the screenshot?

Reviewer: Will do!

     idaho_scheme:
       custom_sample:
         query: --query "(custom_data == 'yes')"

>          max_sequences: 50

Reviewer: We should probably update the screenshot to reflect these changes. I will push a new commit.

Author: Good call! If you don't mind pushing an updated screenshot, that would be great.

Reviewer: Done.
